flow/include/flow/EncryptUtils.h

```diff
@@ -29,22 +29,27 @@
 #include <string>
 #include <string_view>
 
-#define ENCRYPT_INVALID_DOMAIN_ID -1
-#define ENCRYPT_INVALID_CIPHER_KEY_ID 0
-#define ENCRYPT_INVALID_RANDOM_SALT 0
-
 #define AUTH_TOKEN_SIZE 32
 
-#define SYSTEM_KEYSPACE_ENCRYPT_DOMAIN_ID -2
-#define ENCRYPT_HEADER_DOMAIN_ID -3
-
-const std::string FDB_DEFAULT_ENCRYPT_DOMAIN_NAME = "FdbDefaultEncryptDomain";
-
 using EncryptCipherDomainId = int64_t;
 using EncryptCipherDomainNameRef = StringRef;
+using EncryptCipherDomainName = Standalone<EncryptCipherDomainNameRef>;
 using EncryptCipherBaseKeyId = uint64_t;
 using EncryptCipherRandomSalt = uint64_t;
 
+constexpr const EncryptCipherDomainId INVALID_ENCRYPT_DOMAIN_ID = -1;
```

Comment:
```suggestion
constexpr EncryptCipherDomainId INVALID_ENCRYPT_DOMAIN_ID = -1;
```
same below

Reply:
`constexpr` in this context would be `const`, however, I kept it consistent with all other FDB usage, for instance: `fdb_api.hpp`, `blob_granules.hpp`, `TokenSign.cpp` and list goes on. 
I think both usages works fine and should be OK to follow. 
Thanks

Reply:
okay. though I'm wondering whether `constexpr const` will cause the variable to be evaluated at compile time (like `constexpr`) or can be evaluated at runtime (like `const`). For simple literal like this one it should be at compile time anyway.

Reply:
As per C++ [doc](https://en.cppreference.com/w/cpp/language/constexpr) should allow variable/function evaluation at compile time. However, I couldn't find any documentation encouraging/discouraging usage of `const` along with it. 

Again, as I mentioned above it doesn't feel like a big issue one way or the other; will leave it as is; if someone feels strong about it, better to cleanup across the board for uniformity. 
Thanks